src/main.rs

```diff
@@ -128,6 +129,14 @@ fn main() {
                 .default_value("textfile")
                 .takes_value(true),
         )
+        .arg(
+            Arg::new("init-method")
+                .short('m')
```

Comment:
Short 'm' seems weird for initalization method, I suggest we drop the short name.